CHANGELOG.md

```diff
@@ -38,6 +46,11 @@ User-visible changes worth mentioning.
 
 - [#1270] Find matching tokens in batches for `reuse_access_token` option (fix #1193).
 - [#1271] Reintroduce existing token revocation for client credentials.
+
+**[IMPORTANT]** If you rely on being able to fetch multiple access tokens from the same
+client using client credentials flow, you should skip to version 5.3, where this behaviour
+is deactivated by default.
```

Comment:
Now that I wrote this message I realized that I didn't exactly know _what_ I want to point out here, please have a look:

* should I already refer to this as 5.3? Even though it is not yet released and I don't know if it will be called 5.3
* Does the hint make sense at all? I feel this is the best hint I could give, but I had a bad gut feeling writing it.

Reply:
I think it would be 5.2.2

What about the hint... it's hard to say actually...